signac_dashboard/templates/cards/trusted_dict_display.html

```diff
@@ -0,0 +1,5 @@
+<ul>
+  {% for key in escaped_dict %}
+  <li><strong>{{ key }}:</strong> {{ escaped_dict[key] | safe }}</li>
```

Comment:
Let's use the word "safe" or "escaped" instead of "trusted" in this filename. We're using three different terms to denote the same concept.